pages.de/common/dotnet-build.md

```diff
@@ -0,0 +1,32 @@
+# dotnet build
+
+> Baut eine .NET Anwendung und ihre Abhängigkeiten.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/dotnet/core/tools/dotnet-build>.
```

Comment:
We try to keep the links the same in every translation. If you have the correct locale set, you will be forwarded automatically anyways.
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/dotnet/core/tools/dotnet-build>.
```

Reply:
Yes, based on your system locale the links automatically get redirected to translated versions. That's why we don't use l10n Microsoft URLs and instead use generalised ones.